projects/js-packages/components/components/pricing-table/index.tsx

```diff
@@ -165,9 +173,14 @@ const PricingTable: React.FC< PricingTableProps > = ( { title, items, children }
 				</div>
 			</div>
 			<div className={ styles[ 'tos-container' ] }>
-				<Text className={ styles.tos } variant="body-small">
-					{ ToS }
-				</Text>
+				<div className={ styles.tos }>
+					{ showIntroOfferDisclaimer && (
+						<Text variant="body-small">
+							Reduced pricing is a limited offer for the first year and renews at regular price.
```

Comment:
You'll probably want this to be translatable as well.

Reply:
I missed this, thanks!